### ISSUE TITLE
Add tap gestures for "center top" and "center left", etc

### ISSUE THREAD

**Issue by dschil138** [rxhanson/Multitouch-Community#197]
Currently we have corner gestures like this:
<img width="50" alt="corner-tap" src="https://github.com/rxhanson/Multitouch-Community/assets/11950317/fa8a776b-8757-4a54-a9ad-efbcf5ef67aa">

It would be great to have options for the sides like this:

<img width="50" alt="center-tap" src="https://github.com/rxhanson/Multitouch-Community/assets/11950317/e1592024-99a6-43f7-9751-3658789a53f5">

I think these would actually probably be even easier to use than the corners. Especially the top and bottom. I don't think(?) it would trigger to many accidental actions.

**Comment by rxhanson** [comment]
Thanks for the feedback! I tried these out a long time ago when I first added the corner taps and didn't care for them at the time, but will give this another thought when I'm back to reassessing potential new gestures again.

**Comment by dschil138** [comment]
Yeah in playing around with/thinking about the idea more, I could see the bottom tap accidentally being triggered pretty much constantly. 

But the rest of them seem that they would be pretty safe, at least with how I use the trackpad.

I suppose everybody is going to be a little different. It would be really great to have the option tho!
